core/kazoo_endpoint/src/kz_endpoint_recording.erl

```diff
@@ -29,7 +29,8 @@ maybe_record_inbound(FromNetwork, Endpoint, Call) ->
           {'true', {kz_json:path(), kz_json:object()}} | 'false'.
 maybe_record_inbound(_FromNetwork, _Endpoint, _Call, 'undefined') -> 'false';
 maybe_record_inbound(FromNetwork, Endpoint, Call, Data) ->
-    case kz_json:is_true(<<"enabled">>, Data) of
+    case kz_json:is_true(<<"enabled">>, Data)
+        orelse kapps_call:kvs_fetch('record_on_b_leg', 'false', Call) of
```

Comment:
@kageds this seems backwards to me?

If `enabled` is `true` and `record_on_b_leg` is `true`, shouldn't the function return `false` to *not* start the recording on the inbound(caller) leg?